utils/fill_zero.go

```diff
@@ -0,0 +1,111 @@
+package utils
+
+import (
+	"context"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+
+const (
+	EnvFillZeroUtility = "IRONLIB_UTIL_FILL_ZERO"
+	FileMode644        = 0o644 // Owner readwrite, group and others readonly
+	OneMBinBytes       = 1024 * 1024
+	SecondsPerHour     = 3600
+	TotalPercentage    = 100
```

Comment:
We should get rid of these, gomnd isn't really that helpful and will soon go away.

Reply:
Do we need to modify .golangci.yml to disable gomnd?

Reply:
or as you added

`//noline:gomnd // the magic numbers here are fine`

Reply:
removed!